werkit/aws_lambda/test_worker/service.py

```diff
@@ -0,0 +1,10 @@
+import time
+import os
+
+LAMBDA_WORKER_DELAY = 'LAMBDA_WORKER_DELAY'
+env_lambda_worker_delay = int(os.environ[LAMBDA_WORKER_DELAY]) if LAMBDA_WORKER_DELAY in os.environ else 0
```

Comment:
```py
env_lambda_worker_delay = int(os.environ.get('LAMBDA_WORKER_DELAY', 0))
```

Could you include the units in the env var? e.g. `LAMBDA_WORKER_DELAY_SECONDS`?

Reply:
Fixed in 5cbf2f38d0cebf4240abbc12753f29b27fc8e014